packages/nuxt/src/app/composables/fetch.ts

```diff
@@ -140,12 +140,9 @@ export function useFetch<
     controller?.abort?.()
     controller = typeof AbortController !== 'undefined' ? new AbortController() : {} as AbortController
 
-    const isLocalFetch = typeof _request.value === 'string' && _request.value.startsWith('/')
-    let _$fetch = opts.$fetch || globalThis.$fetch
+    const isLocalFetch = typeof _request.value === 'string' && _request.value.startsWith('/') && (!unref(opts.baseURL) || unref(opts.baseURL)!.startsWith('/'))
```

Comment:
```js
useFetch('/users', { $fetch: $fetch.create({ baseURL: 'https://example.com/api/' }) })
// isLocalFetch === true ?
```

Reply:
yes, but it isn't used... will push a perf refactor to avoid computing it at all if not needed...